FBSnapshotTestCase/FBSnapshotTestCasePlatform.m

```diff
@@ -29,3 +30,20 @@ BOOL FBSnapshotTestCaseIs64Bit(void)
   } 
   return [suffixesSet copy];
 }
+
+NSString *FBDeviceAgnosticNormalizedFileName(NSString *fileName)
+{
+  UIDevice *device = [UIDevice currentDevice];
+  CGSize screenSize = [UIScreen mainScreen].bounds.size;
```

Comment:
AFAIR from iOS9 Apple suggests to use UIWindow instead of UIScreen, because of side view / split view. Right now library can't do anything with side view / split view, but this might change in future.